README.md

```diff
@@ -2,3 +2,5 @@
 # Hiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
 # read me please
 # I need a nap
+# bonjour, I'm back
+# you can't stop meeeeeeeeeeeeeeeeee
```

Comment:
you said bonjour, but forgot to say au revoir. how rude of you, please fix your mistake before I accept.

Reply:
too bad i wanted to be rude